swift/apple/FirezoneKit/Sources/FirezoneKit/Stores/Store.swift

```diff
@@ -163,6 +163,13 @@ public final class Store: ObservableObject {
   func beginUpdatingResources(callback: @escaping (ResourceList) -> Void) {
     Log.log("\(#function)")
 
+    guard self.resourcesTimer == nil
```

Comment:
It seems weird to me to use a guard without actually creating a new variable that holds the state you are guarding. If you don't need the resource timer, just make it an `if != nil`.

Reply:
Fixed